README.md

```diff
@@ -295,6 +296,56 @@ clip-retrieval-front 3005
 
 You can also run it with `clip-retrieval front` or back from the python package.
 
+## Clip client
+
+Clip client is a simple python module that allows you to query the backend remotely.
+
+```python
```

Comment:
Nice examples!

Could you also add a API section listing what are the methods available and their parameters?